libpod/container.go

```diff
@@ -270,6 +270,8 @@ type ContainerConfig struct {
 
 	// Network Config
 
+	// NetworkDisabled indicates whether the container uses the network
+	NetworkDisabled bool `json:"networkDisabled"`
```

Comment:
I think we don't need this - we can check if `container.config.Spec.Linux.Namespaces` has a namespace called "net" - if it has none, we have no network

Reply:
Sorry, it's actually `"network"`